java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -129,6 +132,12 @@ public void shutDown() {
 
   @Override
   public void subscribe(Flow.Subscriber<? super String> subscriber) {
+    subscriber.onSubscribe(new Subscription() {
+      @Override
+      public void request(long n) {}
```

Comment:
Why was this change needed?

Reply:
The StepVerifier expects an `onSubscribe` before an `onNext`. I guess it is according to spec?